src/reconstruction/refinement/GPRLogic/Node.m

```diff
@@ -196,9 +198,50 @@ function removeDuplicateLiterals(self)
                 toRemove = setdiff(literals,toKeep);
                 self.children(toRemove) = [];
             end
+        end               
+        
+        function tf = hasNestedChildren(self)
+        % Check, whether this Node has nested children (i.e. whether it has
```

Comment:
These internal functions need proper header formatting.

Reply:
How so?
Whats wrong with the formatting?